src/main/java/com/github/fishio/Preloader.java

```diff
@@ -114,7 +119,7 @@ private static void tryPreLoad(String file, boolean pixelData) {
 		try {
 			image = new Image(file);
 		} catch (Exception ex) {
-			System.err.println("Error while trying to load image " + file);
+			log.log(LogLevel.WARNING, "Error while trying to load image: " + file);
```

Comment:
Chiel, I think that if fails to load the image there, the program should still be able to keep running.

<naming